packages/portal/swagger-ui-web-component/src/element.js

```diff
@@ -162,6 +164,9 @@ export class SwaggerUIElement extends HTMLElement {
       console.warn('For correct positioning, you must enable the sidebar with `has-sidebar="true"` and should only display essentials with `essentials-only="true"`')
     }
 
+    // load swagger UI styles to shadow DOM
+    swaggerUIStyles.use({ target: this.shadowRoot, testId: 'swagger-ui-styles' })
```

Comment:
We should add a test that this test-id exists in `SpecDetails.cy.ts` (we can add it as the last asserting in `it('renders correctly with props')`

Reply:
Added